website/docs/docs/dbt-cloud-apis/project-state.md

```diff
@@ -61,16 +61,20 @@ Most Discovery API use cases will favor the _applied state_ since it pertains to
 
 | Node                                          | Executed in DAG  | Created by execution | Exists in database | Lineage               | States               |
 |-----------------------------------------------|------------------|----------------------|--------------------|-----------------------|----------------------|
-| [Model](/docs/build/models)                   | Yes              | Yes                  | Yes                | Upstream & downstream | Applied & definition |
-| [Source](/docs/build/sources)                 | Yes              | No                   | Yes                | Downstream            | Applied & definition |
-| [Seed](/docs/build/seeds)                     | Yes              | Yes                  | Yes                | Downstream            | Applied & definition |
-| [Snapshot](/docs/build/snapshots)             | Yes              | Yes                  | Yes                | Upstream & downstream | Applied & definition |
+| [Analysis](/docs/build/analyses)   	        | No               | No                   | No                 | N/A 	               | Definition 	      |
```

Comment:
Based on the diagram [here](https://github.com/dbt-labs/docs.getdbt.com/issues/4492#issuecomment-1881658603), _Analysis_ looks to be the same as _Exposure_, which is "Upstream":

```suggestion
| [Analysis](/docs/build/analyses)   	        | No               | No                   | No                 | Upstream            | Definition 	      |
```